packages/expo-store-review/android/build.gradle

```
@@ -0,0 +1,80 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'maven'
+
+group = 'host.exp.exponent'
+version = '8.2.1'
```

```suggestion
version = '2.1.4'
```
Version should match the one from `package.json`

---

Thanks. Wasn’t sure about this as I didn’t find documentation for creating base modules.

---

There's probably one more place with version in `build.gradle`